README.md

```diff
@@ -22,6 +22,27 @@ Prerequisite: [Node.js(LTS version will be fine) and npm](https://docs.npmjs.com
 5. To build and serves the app locally, run `ng serve`. If it's build successfully, there should be a link that look like this: http://localhost:4200/. 
 6. Navigate to the link in your browser, and your should be able to see the app.
 
+## Flight Computer setup Guide
+Prerequisites:
+ - CMake (at least version 3.13)
+ - A compiler:
+   - MacOS - gcc 9 - `brew install gcc` or install through xcode
+   - Linux - gcc 9 - `sudo apt install build-essential`
+   - Arm64 Linux (Pi 4)- gcc 9.3 - Manually install from [here](https://github.com/abhiTronix/raspberry-pi-cross-compilers/wiki/64-Bit-Cross-Compiler:-Installation-Instructions)
+   - Arm32 Linux (Pi 3B+) - gcc 9.3 - Manually install from [here](https://github.com/abhiTronix/raspberry-pi-cross-compilers/wiki/Cross-Compiler:-Installation-Instructions)
+   - Windows - Visual Studio 2019 cl.exe should work fine
```

Comment:
Confirmed works for 2017, however you should mention you need the c++ cmake tools add on for visual studio (not sure if its enabled by default when you install)

Reply:
Yeah it seems like on 2019 when you install Desktop dev with C++ it automatically installs cmake tools, but will specify in case this is different for other versions